### ISSUE TITLE
arris-tg2492lg installs 'tests' package which is forbidden

### ISSUE THREAD

**Issue by onkelbeh** [vanbalken/arris-tg2492lg#16]
Hi,

first I hope you don't mind that I use your integration in [Home Assistant Gentoo Overlay](https://github.com/onkelbeh/HomeAssistantRepository).

**Describe the bug**
During some compile tests I saw that arris-tg2492lg tries to install 'test' package at top level, this is fobidden in Gentoo (and most other Distri's). This could easily be fixed with a small change to setup.py:

For now, I added a small generic `sed` script to my ebuilds:

```bash
src_prepare() {
    sed "s/find_packages()/find_packages(exclude=['tests','tests.*'])/g" -i setup.py || die
    eapply_user
}
```
I fixed this in my repo a few days ago, but I have a reminder for cleanup which just popped up again.
I'll send a pull request in a few seconds.

Thanks a lot.
Please drop me a not when it's fixed, so I can remove the patch.

For the build log, please see https://git.edevau.net/onkelbeh/HomeAssistantRepository/issues/181

 \B.

**Comment by vanbalken** [comment]
Thanks for reporting the issue and the fix. I will let you know when I have released a new version.

**Comment by vanbalken** [comment]
@onkelbeh I've released version 1.0.1 to pypi which includes this fix. Please let me know if this solves your issue.
